src/include/getgrgid_r.c

```diff
@@ -0,0 +1,306 @@
+#define _GNU_SOURCE
```

Comment:
This file probably should have the same comment about coming from musl and if it's not licensed identically to liblxc, the license header should be there too.

Reply:
done